packages/compiler/core/emitter-utils.ts

```diff
@@ -13,6 +15,9 @@ export interface EmitFileOptions {
  * @param options File Emitter options
  */
 export async function emitFile(program: Program, options: EmitFileOptions): Promise<void> {
+  // ensure path exists
+  const outputFolder = path.dirname(options.path);
```

Comment:
I think this should use getDirectoryPath from our own path-utils. Sorry I missed that on first review.